components/authz-service/testhelpers/testhelpers.go

```diff
@@ -99,7 +97,7 @@ func SetupProjectsAndRulesWithDB(t *testing.T) (
 	// TODO(sr): refactor our constructors. Having to maintain the middleware in
 	// three places is tedious and error-prone.
 	serv := connFactory.NewServer(grpc.UnaryInterceptor(
-		grpc_server.InputValidationInterceptor(),
+		server.InputValidationInterceptor(),
```

Comment:
is this in the same package now?

Reply:
Yeah `server/v2` was flattened into `server` which is is where our `InputValidationInterceptor` lived. It previously had this interesting input label:

```
grpc_server "github.com/chef/automate/components/authz-service/server"
```